app/ledger.js

```diff
@@ -294,7 +294,7 @@ const doAction = (state, action) => {
 var init = () => {
   try {
     initialize(getSetting(settings.PAYMENTS_ENABLED))
-  } catch (ex) { console.log('ledger.js initialization failed: ' + ex.toString() + '\n' + ex.stack) }
+  } catch (ex) { console.error('ledger.js initialization failed: ' + ex.toString() + '\n' + ex.stack) }
```

Comment:
This one could also just log the raw object `ex` instead of `ex.stack` since we'll already get the stack trace. Maybe something like:
`console.error('ledger.js initialization failed: ', ex)`